remove passive voice and screenshots

nataliejeremy opened the pull request:
closes #828 

This PR:
Removes screenshots
Removes passive voice
Includes general edits. 

There are two Gandi docs. I vote for keeping this one:https://pantheon.io/docs/articles/sites/domains/gandi-pantheon-pointing-your-dns/ and deleting https://pantheon.io/docs/articles/sites/domains/setting-up-a-domain-with-gandi/ but would like to get feedback first from @ari-gold and @bmackinney

ari-gold commented:
@nataliejeremy I don't have any experience with Gandi. My only question is if we need to maintain our own docs for them or if we can link to Gandi's documentation?

nataliejeremy commented:
Doesn't get much traffic. @bmackinney what do you think? 
![screen shot 2015-08-26 at 12 19 09 pm](https://cloud.githubusercontent.com/assets/9138609/9503590/bda982a6-4bec-11e5-9f16-0a94b2e00a51.png)

nataliejeremy commented:
- Deleted setting-up-a-domain-with-gandi.md and added a redirect to /docs/articles/sites/domains/gandi-pantheon-pointing-your-dns/
- Confirmed no existing docs point to the deleted article

ari-gold commented on source/docs/articles/sites/domains/gandi-pantheon-pointing-your-dns.md:
````diff
@@ -6,41 +6,40 @@ category:
 ```
+
+<div class="alert alert-warning" role="alert">
+<h4>Note</h4>
````
This is outdated. See our main domains/DNS doc

nataliejeremy replied:
@ari-gold Is this correct? 
Instead, add CNAME records for both the bare domain (@) and the FQDN (www), both pointed to live-example.pantheon.io.
Taken from https://pantheon.io/docs/articles/sites/domains/#serving-sites-from-bare-domains-with-http

ari-gold replied:
@nataliejeremy 
No, it looks like Gandi doesn't allow adding a CNAME for the bare domain: 

> When not to use a CNAME
>You cannot create a CNAME record for a bare domain. It can only be on a subdomain.
from https://wiki.gandi.net/en/dns/zone/cname-record

Ping me in Slack or the office if you want to discuss.

ari-gold commented on source/docs/articles/sites/domains/gandi-pantheon-pointing-your-dns.md:
```diff
@@ -6,41 +6,40 @@ category:
-<h4>Note</h4>
-Be sure to include the trailing "."</div>
+
2. Next, change the www CNAME to match the following:
```
This is outdated. See our main domains/DNS doc

nataliejeremy commented:
@ari-gold Thanks, I updated that info.

rachelwhitton commented:
## Redirect Needed

[setting-up-a-domain-with-gandi](https://pantheon.io/docs/articles/sites/domains/setting-up-a-domain-with-gandi/) > [gandi-pantheon-pointing-your-dns/](https://pantheon.io/docs/articles/sites/domains/gandi-pantheon-pointing-your-dns/)
